site/_posts/2021-04-23-announcing-fusionauth-1-26.md

```diff
@@ -0,0 +1,103 @@
+
+* Create an "API" entity type with the following permissions: `query`, `execute`, and `configure`.
+* Create an "reminder API" entity
+* Set up a "todo API" entity
+* Grant the reminder API `query` permissions on the todo API
+
+When the reminder API is ready to gather upcoming tasks, it "signs in" with the Client Credentials grant. FusionAuth will, if the reminder API has the appropriate permission on the todo API, create a JWT. The reminder API can present the JWT, perhaps as a bearer token, to the todo API. The todo API can then process the request and return the needed data.
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/list-of-entities.png" alt="A list of APIs created using Entity Management." class="img-fluid" figure=false %}
+
+All of this set up and execution can be done via API, making Entity Management a great way to model machine to machine communication in a dynamic service oriented application, an internet of things system, or elsewhere. You don't have to use the grant mechanism either. If you want to model relationships in FusionAuth for any reason, store JSON arbitrary data in the `entity.data` field. Read more about [Entity Management here](/docs/v1/tech/core-concepts/entity-management/).
+
+## An API to manage API keys
+
+In true Inception style, there is now an API to manage FusionAuth API keys. This was a community request last year. You can create, copy, update, and delete API keys. These keys are indistinguishable from API keys created in the user interface. 
+
+They can be restricted in a number of ways:
+
+* By tenant
+* By endpoint
+* By permission on an endpoint
+
+This feature allows interesting automation. For example, you can now generate API keys with very specific permissions, distribute and use them as needed, and then destroy them.
+
+## Other enhancements
+
+Some of the other enhancements included in this release are:
+
+* Support for FusionAuth acting as an SP for IdP initiated SAMLv2 SSO (this is a premium feature).
+* An extension to the user registration forms to allow for a user profile self service page (this is a premium feature).
+* A Prometheus endpoint for monitoring.
+* Support for licensing in air-gapped deployments.
+* The login success and failure webhook events now include the IP address.
+
+Also, there's a new themeable FusionAuth index page so when your users accidentally arrive at `auth.example.com`, they are not forwarded to the FusionAuth admin UI login page. (To proceed to that login page when using the default theme, click the lock in the upper right.)
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/themable-index-page.png" alt="The themable root page. No more users ending up on the login page." class="img-fluid" figure=false %}
+
+Like I said, this was a big release!
+
+## Bugs squashed
+
+There were a couple of bugs fixed, including:
+
+* A registration for an inactive application previously couldn't be deleted; now it can be.
```

Comment:
Previously, a registration for an inactive application couldn't be deleted -- now it can!